### PR TITLE
Add feedback when card is missing

### PR DESCRIPTION
**Pull request by thomashbrnrd**

### CODE REVIEW

File: frontend/src/main.css [hunk 1]
```diff
@@ -35,18 +31,6 @@ a {
...
-  top: 97%;
-}
-
-button {
-  white-space: nowrap;
-  display: inline-flex;
-  width: fit-content;
-  align-items: center;
-  gap: 0.5rem;
-}
```
Comment (leihuayi) [inline frontend/src/main.css:48]: are you sure removing this piece of code does not have any influence anywhere in the display of all buttons in the app ?

**Review by leihuayi — CHANGES_REQUESTED** [review]
- the name of the branch has no relationship whatsoever with what it actually does
- this work should be placed along with the global refactoring planned with @nutfdt 

Please rename this branch and do the complete refactoring work in it.

**Review by leihuayi — APPROVED** [review]
[EDIT] some changes necessary, see below

File: frontend/src/components/ResultPage.vue [hunk 2]
```diff
@@ -119,131 +148,111 @@ function sendFeedback (isCorrect: boolean) {
...
+            <DsfrTag
+              class="fr-tag--sm error-tag"
+              label="Indice de fiabilité insuffisant"
+            />
+          </div>
+          <p class="category fr-callout__title mt-3">
+            <img
+              class="px-2"
+              src="@/assets/guide-identification/icones/gun.jpg"
+              alt=""
```
Comment (leihuayi) [inline frontend/src/components/ResultPage.vue:165]: Update this design when confidence level is low

File: frontend/src/components/ResultPage.vue [hunk 3]
```diff
@@ -119,131 +148,111 @@ function sendFeedback (isCorrect: boolean) {
...
-                  class="fr-icon-warning-fill text-blue mx-1"
-                  aria-hidden="true"
-                /></span>
-              </div>
-              <p class="text-sm text-justify">
-                Le résultat donné par Basegun n’emporte qu’une simple <span class="font-bold">valeur de renseignement</span>.
-                Pour faire référence dans une procédure, il <span class="font-bold">doit impérativement et réglementairement être validé</span>
+              <p v-html="disclaimer" />
+            </div>
+            <MissingCardAlert v-if="MEASURED_GUNS_TYPOLOGIES.includes(typology) && isCardDetected === false" />
```
Comment (leihuayi) [inline frontend/src/components/ResultPage.vue:228]: do not display this alert if it is confirmed that it is a dummy weapon

**Review by leihuayi — CHANGES_REQUESTED** [review]

File: frontend/src/components/ResultPage.vue (hunk 2 above)
Comment (thomashbrnrd) [reply frontend/src/components/ResultPage.vue]: Done ! 
image

**Review by leihuayi — APPROVED** [review]

**Pull request merged**